src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 spring.application.name=ExamNinja
 # MySQL Configuration
-spring.datasource.url=jdbc:mysql://localhost:3306/exam_ninja
+spring.datasource.url=jdbc:mysql://localhost:3306/examninja_db
```

Comment:
Assume that this application will be hosted on cloud. Then this property localhost will not work. You need to think of how the URL will be in case it happens to be a database server hosted on cloud.

Reply:
Will add a sample URL for running in cloud platform.

Reply:
addressed and provided generic format